.rubocop_todo.yml

```diff
@@ -176,26 +163,26 @@ Style/SymbolProc:
   Exclude:
     - 'lib/spotlight/dor/indexer.rb'
 
-# Offense count: 5
+# Offense count: 2
 # Cop supports --auto-correct.
 # Configuration parameters: EnforcedStyle, SupportedStyles.
 Style/TrailingBlankLines:
   Exclude:
-    - 'lib/spotlight/dor/indexer.rb'
-    - 'spec/integration/gdor_integration_spec.rb'
-    - 'spec/models/spotlight/resources/purl_spec.rb'
     - 'spec/models/spotlight/resources/searchworks_spec.rb'
     - 'spec/spec_helper.rb'
 
-# Offense count: 44
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyleForMultiline, SupportedStyles.
+Style/TrailingComma:
+  Exclude:
+    - 'spec/unit/spotlight/dor/indexer_spec.rb'
```

Comment:
Off-hand, I don't see what introduced this violation, but if it's an easy fix...